roles/wordpress-setup/templates/wordpress-site.conf.j2

```diff
@@ -46,6 +47,11 @@ server {
     set $skip_cache 1;
   }
 
+  # Update an expired cache item in the background, serve stale cache to client.
+  {% if item.value.cache.enabled and item.value.cache.background_update %}
+    fastcgi_cache_background_update on
```

Comment:
forgot the semi-colon at the end

Reply:
Also should this be in the cache_config block?

Reply:
yeah I'd prefer this to be grouped in `cache_config` 👍